ArduPlane/quadplane.cpp

```diff
@@ -1926,6 +1941,10 @@ void QuadPlane::update_throttle_hover()
  */
 void QuadPlane::motors_output(bool run_rate_controller)
 {
+#if HAL_WITH_ESC_TELEM
+    motors->takeoff_check(takeoff_rpm_min, takeoff_rpm_max, false);
```

Comment:
the land_complete false is a problem, and means this doesn't work at all.
We need this to only impact automatic takeoff, not manual takeoff or a back transition. If we lose RPM monitoring on a motor we must not fail to spool up on back transition

Reply:
can use in_vtol_takeoff()